File: tests/pyconverter-test/cases/class_super.py
class Base:
    def __init__(self, num):
        pass

    def instanceM(self):
        print(1)
    def staticM():
        print(2)

class Sub(Base):
    def __init__(self):
        super().__init__(5)

    def instanceM(self):
        super().instanceM()
        print(3)
    def staticM():
        print(4)


x = Base(4)
y = Sub()

File: tests/pyconverter-test/baselines/class_super.ts
class Base {
    constructor(num: number) {
        
    }
    
    public instanceM() {
        console.log(1)
    }
    
    public static staticM() {
        console.log(2)
    }
    
}

class Sub extends Base {
    constructor() {
        super(5)
    }
    
    public instanceM() {
        super.instanceM()
        console.log(3)
    }
    
    public static staticM() {
        console.log(4)
    }
    
}

let x = new Base(4)
let y = new Sub()
